devise-jwt.gemspec

```diff
@@ -22,14 +22,14 @@ Gem::Specification.new do |spec|
   spec.require_paths = ["lib"]
 
   spec.add_dependency 'devise', '~> 4.0'
-  spec.add_dependency 'warden-jwt_auth', '~> 0.10'
+  spec.add_dependency 'warden-jwt_auth', '~> 0.11'
 
   spec.add_development_dependency "bundler", "> 1"
   spec.add_development_dependency "rake", "~> 13.0"
   spec.add_development_dependency "rspec"
   # Needed to test the rails fixture application
   spec.add_development_dependency 'rails', '~> 6.0'
-  spec.add_development_dependency 'sqlite3', '~> 1.3'
+  spec.add_development_dependency 'sqlite3', '1.5.4'
```

Comment:
Fixed at last 1.5.x because newer versions require Ruby 3.x to compile on Apple Silicon.